addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -436,6 +436,50 @@ QUnit.module("Fields", (hooks) => {
         await click(target);
     });
 
+    QUnit.test(
+        "widget many2many_tags_avatar add/remove tags in kanban view",
+        async function (assert) {
+            assert.expect(3);
+
+            await makeView({
+                type: "kanban",
+                resModel: "turtle",
+                serverData,
+                arch: `
+                <kanban>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div class="oe_kanban_global_click">
+                                <field name="display_name"/>
+                                <div class="oe_kanban_footer">
+                                    <div class="o_kanban_record_bottom">
+                                        <div class="oe_kanban_bottom_right">
+                                            <field name="partner_ids" widget="many2many_tags_avatar"/>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>`,
+                async mockRPC(route, { method, args }) {
+                    if (method === "write") {
+                        assert.step(`write_${args[1].partner_ids[0][2]}`);
+                    }
+                },
+            });
+            const o_kanban_record = target.querySelector(".o_kanban_record");
+            await click(o_kanban_record, ".o_quick_assign", true);
+            const popover = document.querySelector(".o_popover_container");
+
+            // add and directly remove an item
+            await click(popover.querySelector(".o-autocomplete--dropdown-item"));
+            await click(popover.querySelector(".o_tag .o_delete"));
```

Comment:
It s risky if your popover is rerender and you have anoter html element.
It s more robust to do targertquerySelector(".o_popover_container ....") at each time